packages/charts/src/core/grid/grid.ts

```diff
@@ -245,16 +245,22 @@ export abstract class Grid implements IGrid {
      */
```

Comment:
These changes fix the 1px tall area at the top of the linear gauge where the mouse interactive area was 1px taller than the gauge.